src/components/classic/documentation/testing-changes-locally.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: default_md
+title: Testing changes locally 
+title-class: page-title-classic
+type: classic
+---
+
+[Developers](developers) > [Testing changes locally](testing-changes-locally)
+
+Testing changes locally
+-----------------------------
+
+### Broker
+After running existing or newly added unit tests, if you want to do integration testing on your change, you can [build](building) the repo locally to generate the release bundle and then unzip the bundle to test broker behaviour. 
```

Comment:
I'd expect a page about 'testing changes locally' to actually actually cover something about the tests, or at the very least reference to some other existing content that does speak to _running tests_, rather than gloss over it with simply "After running existing or newly added unit tests".

Maybe the page title needs a rethink. This seems more about 'running an example with your locally-built changes'.